docker-compose.yaml

```
@@ -61,13 +72,18 @@ services:
   objectiv_notebook:
     container_name: objectiv_notebook
     # default image: eu.gcr.io/objectiv-production/notebook:latest
-    image: ${OBJECTIV_CONTAINER_URL-eu.gcr.io/objectiv-production}/notebook:${OBJECTIV_CONTAINER_TAG-latest}
+    image: ${OBJECTIV_CONTAINER_URL-objectiv}/notebook:${OBJECTIV_CONTAINER_TAG-latest}
     ports:
-      - 127.0.0.1:8888:8888
+      - "8888:8888"
+      - "8053:8053"
```

```suggestion
      - "127.0.0.1:8888:8888"
      - "127.0.0.1:8053:8053"
```
the token is hardcoded, so we don't want to expose the notebook to the outside world